code/modules/mining/equipment_locker.dm

```diff
@@ -592,7 +590,9 @@
 		var/pressure = environment.return_pressure()
 		if(pressure < 50)
 			name = "strong resonance field"
-			resonance_damage = 50
+			resonance_damage = 200
```

Comment:
How can you justify this much damage when the kinetic accelerator only does 40 damage and is slower? Both weapons do not require a melee attack.